aten/src/THCUNN/generic/VolumetricDilatedMaxPooling.cu

```diff
@@ -163,7 +163,7 @@ void THNN_(VolumetricDilatedMaxPooling_updateOutput)(
     inputHeight = THCTensor_(size)(state, input, 2);
     inputWidth  = THCTensor_(size)(state, input, 3);
   }
-  else if (THCTensor_(nDimension)(state, input) == 5)
+  else /* 5D */
```

Comment:
Do we have any checks on the dimensionality of inputs here? I can't find them

Reply:
Aren’t they in the shapeCheck function above?